valhalla/baldr/graphtile.h

```diff
@@ -882,16 +882,16 @@ class GraphTile {
   std::size_t lane_connectivity_size_{};
 
   // Predicted speeds
-  PredictedSpeeds predictedspeeds_;
+  PredictedSpeeds predictedspeeds_{};
 
   // Map of stop one stops in this tile.
-  std::unordered_map<std::string, GraphId> stop_one_stops;
+  std::unordered_map<std::string, GraphId> stop_one_stops{};
```

Comment:
did this give a warning or? `unordered_map` shouldn't have problems with not being initialized

Reply:
Yeah, clang-tidy or something was giving me linter warnings 🤷